R/class-AbundanceDensityPlot.R

```diff
@@ -229,13 +229,49 @@ setMethod(".multiSelectionResponsive", "AbundanceDensityPlot",
         return(FALSE)
 })
 
+#' @importFrom methods callNextMethod
+#' @importFrom iSEE .getEncodedName .addTourStep
+setMethod(".definePanelTour", "AbundanceDensityPlot", function(x) {
```

Comment:
keep lines shorter than 50 characters and use spaces multiple of 4, here and everywhere else